client/lib/post-normalizer/index.js

```diff
@@ -101,6 +101,12 @@ function maxWidthPhotonishURL( imageURL, width ) {
 		return imageURL;
 	}
 
+	if ( ! ( endsWith( parsedURL.host, 'wp.com' ) ||
```

Comment:
Perhaps we could pop these domain endings into an array rather than doing three separate comparisons?